travis.sh

```diff
@@ -34,19 +34,24 @@ else
     XS="$(pkg xs)"
 fi
 
-if [ "${EXTRA_REMOTES}" = 1 ]; then
+if [ ! -z "${EXTRA_REMOTES}" ]; then
     opam remote add extra "$EXTRA_REMOTES"
 fi
 
 if [ "${OPAM_LINT}" = 1 ]; then
     find packages -iname opam -print | xargs -n 1 opam lint
 else
     opam install -y depext
-    opam depext  -y $UPSTREAM $XS
-    opam install -y -j 4 $UPSTREAM $XS
-    # Workaround to mark failed uninstall as error. We only test
-    # the uninstall of the xs packages but not of the upstream packages.
-    opam remove  -y $XS
-    opam install -y -j 4 $XS
+    if [ "${OCAML_VERSION}" = "4.02" ]; then
```

Comment:
For newer compilers I think it's more important to know if we can actually compile xapi packages. Also, uninstallation should be working (or broken) independently of the compiler version.